### DISCUSSION TITLE
Aufgabe 5.

### DISCUSSION THREAD

**Discussion by nezrinstyles** [nezrinstyles/Gruppe-4#10] [General]
Da wir mit allen 4 Aufgaben fertig sind, können wir endlich hier mit einer Diskussion unserer Ergebnisse fortfahren.

**Comment by xi-wa** [comment]
Ich hab gerade gesehen, dass es Aufgabe 4 zweimal gibt
Welche ist die richtige?

**Reply by YuxinZhuang** [reply]
die zwei sind gleich,oder kann ich eine löschen, und muss ich die Bilder in (a)+(b) hinfügen, um günstig zu diskutieren?

**Reply by HongmingQi** [reply]
> Ich hab gerade gesehen, dass es Aufgabe 4 zweimal gibt
> Welche ist die richtige?

Wenn ich den Zweig in den Hauptzweig zusammenführe, gibt es zwei Aufgabe4s, ich bin mir nicht ganz sicher, warum das passiert.😫

**Reply by nezrinstyles** [reply]
Schauen Sie bitte welche von beiden richtige ist, da morgen sollen wir schon mit alles fertig sein und den Link abgeben.

**Reply by xi-wa** [reply]
Und was machen wir mit den Funktionen?
Das haben wir ja auch zwei mal

**Reply by nezrinstyles** [reply]
Ich habe eigentlich nicht verstanden warum haben wir die Funktionen zwei mal

**Comment by YuxinZhuang** [comment]
ich habe eine gelöscht, und wir haben echt die Funktion R Skript 1 ein bisschen geändert,obwohl sich der Code ein wenig geändert hat, bleibt der allgemeine Inhalt derselbe, was es uns erleichtert, visuelle Diagramme zu erstellen.

**Reply by nezrinstyles** [reply]
okay gut

**Reply by xi-wa** [reply]
dann müssen wir die Funktionen zu den aus "WA Github.R" umändern, damit wir nur ein Skript mit Funktionen haben

**Reply by nezrinstyles** [reply]
Du hast recht

**Comment by nezrinstyles** [comment]
Also ja, besprechen wir dann auch unsere Ergebnisse

**Comment by xi-wa** [comment]
Soll sich jeder die Grafiken selber mit den Code aus Aufgabe 4 ausgeben lassen oder werden die anderen drei es hochladen?
Für mich ist beides ok

**Reply by YuxinZhuang** [reply]
aber für(e)(f) ein Grafik schwer ausmachen, glaube ich

**Reply by xi-wa** [reply]
Also was machen wir dann?

**Reply by YuxinZhuang** [reply]
ich habe in A4 die Ergebnisse mit R Skript Funktion 2 von(a) bis (d) geschrieben,aber (e) (f) errror macht, haben ihr andere Idee:/

**Reply by aYajie** [reply]
Ja,ich habe das gleiche Problem, es wird ständig "error" angezeigt...

**Reply by xi-wa** [reply]
Habe es gerade ausprobiert, zu (e) bekomme ich allerdings etwas raus.
Probiert vielleicht mal folgendes:
E(df, 4, 3) und E(df, 5, 3)

ich weiß allerdings nicht ob der Output das ist, was ihr euch vorgestellt habt

**Reply by HongmingQi** [reply]
> Habe es gerade ausprobiert, zu (e) bekomme ich allerdings etwas raus.
> Probiert vielleicht mal folgendes:
> E(df, 4, 3) und E(df, 5, 3)

<img width="831" alt="截屏2023-02-28 20 31 10" src="https://user-images.githubusercontent.com/125801280/221959117-f422ed39-1a00-4877-8c72-35dec8b328db.png">

**Reply by HongmingQi** [reply]
> > Habe es gerade ausprobiert, zu (e) bekomme ich allerdings etwas raus.
> > Probiert vielleicht mal folgendes:
> > E(df, 4, 3) und E(df, 5, 3)
> 
> <img alt="截屏2023-02-28 20 31 10" width="831" src="https://user-images.githubusercontent.com/125801280/221959117-f422ed39-1a00-4877-8c72-35dec8b328db.png">

Vielleicht ist dieser Effekt richtig?

**Reply by aYajie** [reply]
Aber was können wir aus diesem Ergebnis schließen...? Ich verstehe das nicht wirklich...

**Reply by xi-wa** [reply]
Lass dass lieber in der anderen Diskussion besprechen, das hier ist für Aufgabe 5

**Comment by YuxinZhuang** [comment]
[Rplot.pdf](https://github.com/nezrinstyles/Gruppe-4/files/10854887/Rplot.pdf)
Hier ist die Histogramm von Studienfach  und Mathe-LK

**Reply by YuxinZhuang** [reply]
Entschuldigung, ich habe falsch gesagt,das Bild zeigt uns das Histogramm von 6 deskriptiven Statitik mit drei Variable.

**Reply by aYajie** [reply]
Den größten Variationskoeffizienten habe ich für "Interesse.an.Mathe" gefunden.

Ein größerer Variationskoeffizient bedeutet, dass die Daten eine größere Variabilität aufweisen. Dies kann darauf hindeuten, dass die Variablen einen größeren Wertebereich haben oder dass es mehr Extremwerte im Datensatz gibt.

"Alter" hat den kleinsten Variationskoeffizienten, was darauf hinweist, dass die Datenpunkte näher am Mittelwert liegen und die Variabilität der Daten geringer ist.

**Reply by YuxinZhuang** [reply]
die sd und Varianz von drei Variable sind fast gleich ,das bedeutet ,dass die Daten sind durchschnittlich und hat geringe Schwankung

**Reply by aYajie** [reply]
Das denke ich auch,vielleicht haben andere noch mehr Ideen?

**Comment by xi-wa** [comment]
Da Aufgabe 4 nun fertig ist, können wir ja mit der Diskussion beginnen.
Ist euch bei der Auswertung etwas auffälliges oder interessantes aufgefallen?

**Reply by YuxinZhuang** [reply]
ich habe intuitiv bemerkt,bei mean, Quantil1, Quantil 2 sind das Alter fast gleich

**Reply by HongmingQi** [reply]
> ich habe intuitiv bemerkt,bei mean, Quantil1, Quantil 2 sind das Alter fast gleich

Aus dem Histogramm geht hervor, dass die kumulative Häufigkeit der statistischen Fächer hoch ist; könnte dies ein Hinweis auf eine hohe Bereitschaft sein, Statistik zu wählen?

**Reply by xi-wa** [reply]
Ich glaub es ist wichtiger den Zusammenhang zwischen Studienfächer und den anderen Merkmalen zu betrachten

**Comment by aYajie** [comment]
[Rplot01.pdf](https://github.com/nezrinstyles/Gruppe-4/files/10854938/Rplot01.pdf)
Hier ist das Histogramm für Häufigkeit von Studienfach und  Mathe-LK

**Reply by nezrinstyles** [reply]
sieht sehr gut an

**Comment by xi-wa** [comment]
Bei der Funktion (c) ist auch zu erkenne, dass der Studiengang Mathe beispielsweise nur Werte zwischen 5 und 7 annimmt bezüglich "Interesse an Mathe" und der Studiengang Informatik nur Werte zwischen 1 und 5

**Reply by nezrinstyles** [reply]
Wollte eigentlich gerade darüber schreiben, habe das auch bemerkt

**Reply by xi-wa** [reply]
Das arith. Mittel von "Interesse an Mathe/Programmieren" ist sich auch ziemlich ähnlich und liegt etwas höher als 3,5 
Also kann man sagen, dass die Interesse an Mathe und Programmieren der Studiengänge sich eher zu "hohe Interesse" lehnt

**Reply by YuxinZhuang** [reply]
du hast echt Recht

**Reply by aYajie** [reply]
Ja,ich finde das auch.Obwohl "Interesse.an.Mathe" etwas niedriger ist als "Interesse.an.Programmieren", ist es fast identisch.

**Reply by nezrinstyles** [reply]
Bei der deskriptive statistiken ist Prozentsatz für Interesse an Programieren eigentlich niedrig, also 4% nur

**Reply by nezrinstyles** [reply]
Und für Interesse an Mathe sogar 7%

**Reply by xi-wa** [reply]
Etwas seltsam, wenn man sich nämlich die Kreuztabelle zu beiden anschaut, ist die Anzahl an Personen, die "7" gewählt haben deutlich höher  als 7% oder 4%

**Reply by nezrinstyles** [reply]
Ja, hab es auch jetzt gesehen

**Comment by HongmingQi** [comment]
Kommentare zu Historgramm für t.test p-Value?

**Reply by HongmingQi** [reply]
> Kommentare zu Historgramm für t.test p-Value?

[](url)
[Rplot02 (1).pdf](https://github.com/nezrinstyles/Gruppe-4/files/10855209/Rplot02.1.pdf)

**Reply by nezrinstyles** [reply]
wir können den wesentlichen Unterschied zwischen Mathematik und Programmierung deutlich erkennen

**Reply by nezrinstyles** [reply]
aber soll age eigentlich nicht bei der Histogramm sein

**Reply by YuxinZhuang** [reply]
Ja,ich habe falsch gemacht, ihr könnt das Alter nicht berücsichtigen

**Reply by HongmingQi** [reply]
Ich habe eine Frage, ob dieses Bild zeigt , dass die Wahrscheinlichkeit, dass interest_programming unter der Nullhypothese auftritt, kleiner ist als die beiden anderen?

**Reply by YuxinZhuang** [reply]
vielleicht können wir ttest weg,das ist nicht wichtig

**Comment by xi-wa** [comment]
Interessant wäre noch zu beachten, dass die Häufigkeiten für Mathe - LK ungefähr zu 50% bei ja und nein liegen, obwohl die Studiengänge sehr viel Mathe beinhalten

**Reply by YuxinZhuang** [reply]
jaaaa, hab ich auch gemerkt

**Reply by aYajie** [reply]
Ja, es gibt einen leichten Unterschied, aber er beträgt immer noch etwa 50 %.

**Reply by xi-wa** [reply]
Schaut man sich die Kreuztabelle an ist zu erkennen, dass "Mathe" deutlich öfters zum Merkmal "Mathe-LK" ja gewählt haben, bei Data Science und Informatik ist es etwa gleich und bei Statistik schwankt es eher zu "nein".
Man kann daraus schließen, dass die Wahrscheinlichkeit hoch ist, dass man Mathe LK hatte, wenn man Mathe studiert

**Reply by nezrinstyles** [reply]
Allerdings ist es wichtig zu beachten, dass diese Beobachtung nicht zwangsläufig auf jeden einzelnen Studierenden zutrifft.

**Comment by xi-wa** [comment]
Sollen wir den Zusammenhang schonmal aufdecken?

**Reply by nezrinstyles** [reply]
Ja, ich denke es ist schon Zeit

**Reply by nezrinstyles** [reply]
Ich möchte euch alle auch für die gute Zusammenarbeit danken.

**Reply by xi-wa** [reply]
Also Nazrin und ich haben es bei "Interesse an Programmieren" so codiert, sodass jene vom Studiengang "Informatik" eine höhere Interesse haben als die anderen Studiengänge haben.
Die Interesse vom Studiengang "Data Science" sollte niedriger als "Informatik" liegen aber höher als "Mathe" und "Statistik"

**Reply by YuxinZhuang** [reply]
ich danke euch alle auch:)

**Reply by aYajie** [reply]
Ja, ich möchte mich auch bei Ihnen für Ihre Hilfe bedanken :)

**Reply by nezrinstyles** [reply]
> Also Nazrin und ich haben es bei "Interesse an Programmieren" so codiert, sodass jene vom Studiengang "Informatik" eine höhere Interesse haben als die anderen Studiengänge haben. Die Interesse vom Studiengang "Data Science" sollte niedriger als "Informatik" liegen aber höher als "Mathe" und "Statistik"

Ja genau, die Interesse an Mathe von Mathematik Studierenden ist höher als bei anderen, und bei der Interesse an Programmierung ist es dasselbe Situation.

**Reply by xi-wa** [reply]
Des Weiteren hatten Mathe Studenten eine höhere Wahrscheinlichkeit für "Mathe-LK"
Der Rest lag bei 60% - ja und 40% - nein
Deshalb hat es mich auch ein bisschen verwundert, dass es bei der Auswertung eine 50/50 Aufteilung war

**Reply by aYajie** [reply]
> Des Weiteren hatten Mathe Studenten eine höhere Wahrscheinlichkeit für "Mathe-LK" Der Rest lag bei 60% - ja und 40% - nein Deshalb hat es mich auch ein bisschen verwundert, dass es bei der Auswertung eine 50/50 Aufteilung war

Ah,vielleicht gab es also einen kleinen Fehler in der Analyse...

**Reply by nezrinstyles** [reply]
Hm, ich habe es zuerst gar nicht bemerkt

**Comment by HongmingQi** [comment]
Um ehrlich zu sein, haben Sie eine sehr wichtige Rolle bei dieser Zusammenarbeit gespielt, und ich bin Ihnen für Ihr Engagement dankbar.
